packages/vscode-mdx/src/document-drop-edit-provider.js

```diff
@@ -41,6 +68,33 @@ export const documentDropEditProvider = {
       }
     }
 
+    if (uriListItem) {
+      const value = await uriListItem.asString()
+      const uris = value.split(/\r?\n/)
+      /** @type {string[]} */
+      const content = []
+
+      for (const line of uris) {
+        const uri = Uri.parse(line)
```

Comment:
Could the URI list also include comments or titles? https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Recommended_drag_types

Reply:
Apparently, but I haven’t encountered them while testing. I now skip over lines that are invalid URLs.